src/adapter/src/session.rs

```diff
@@ -555,11 +553,10 @@ impl<T: TimestampManipulation> Session<T> {
 
     /// Resets the session to its initial state. Returns sinks that need to be
     /// dropped.
-    pub fn reset(&mut self) -> Vec<ComputeSinkId> {
-        let (drop_sinks, _) = self.clear_transaction();
+    pub fn reset(&mut self) {
+        let _ = self.clear_transaction();
```

Comment:
This doesn't clear the session's transaction now.

Reply:
I think maybe you're mixing up the `clear_transaction` in `session.rs` with the `clear_transaction` in `sql.rs`. This is the one in `session.rs` which does clear the session's transaction.

Or I'm misunderstanding this comment.